pkg/artifactory/util.go

```diff
@@ -136,6 +136,14 @@ func mergeMaps(schemata ...map[string]interface{}) map[string]interface{} {
 	}
 	return result
 }
+
+func copyInterfaceMap(source map[string]interface{}, target map[string]interface{}) map[string]interface{} {
```

Comment:
You could have literally copied and changed the `mergeSchema` function so you could any number of maps.

Reply:
#gosucks

Reply:
Oh yeah, and here you are mutating the target map.